src/Files.Uwp/Strings/en-US/Resources.resw

```diff
@@ -2792,4 +2789,13 @@ We use App Center to track which settings are being used, find bugs, and fix cra
   <data name="ReviewFilesContent" xml:space="preserve">
     <value>Would you like to review Files?</value>
   </data>
+  <data name="SetAsBackgroundFlyout" xml:space="preserve">
+    <value>Set as background</value>
+  </data>
+  <data name="SetAsLockscreen" xml:space="preserve">
+    <value>Set as lockscreen background</value>
+  </data>
+  <data name="SetAsBackground" xml:space="preserve">
+    <value>Set as desktop background</value>
```

Comment:
```suggestion
    <value>Desktop</value>
```

Reply:
Should the Slideshow one be changed too?

Reply:
I think slideshow should stay as is. Just verifying, when multiple images are selected, the only option shown is the slideshow button, is that correct?

Reply:
No, it currently shows all options, but it will only apply the function to the first selected. I can change that though.